### PR TITLE
Add networking and binaries sections

### PR DESCRIPTION
**Pull request by h3ndrk**
What it says, in addition some small tweaks in other sections. Maybe there is more that I forgot...

Feel free to discuss the changes.

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -51,6 +51,16 @@ _How application logic is executed._
 </summary>
 </details>
 
+<details><summary><b>Networking & Protocols</b></br>
```
Comment (JSydll) [inline Readme.md]: I think that's worth a dedicated "main" section - I wouldn't say it's a core concept or principle in itself. Maybe `1.7`? We can of course also first merge and then reorder :+1:

File: Readme.md [hunk 2]
```diff
@@ -145,6 +155,8 @@ _How to use built-in features to communicate between processes._
 * sockets
 * pipes
 * shared memory API (`shm_open`, `mmap`, ...)
+* message queues
```
Comment (JSydll) [inline Readme.md:148]: Good catch!

File: Readme.md [hunk 3]
```diff
@@ -243,6 +255,18 @@ _How to implement security features with existing libraries._
 * `openssl`
 * `libsodium`
 * `wolfssl`
+* Mbed TLS
+</details>
+
+<details><summary><b>Binaries</b></br>
```
Comment (JSydll) [inline Readme.md:263]: I was unsure about this as I first thought it rather to be related to application development... But you're right, it's quite Linux-specific.

**Review by JSydll — APPROVED** [review]
Aside from moving sections around, this looks like a nice extension to me. Thanks for contributing!

File: Readme.md (hunk 1 above)
Comment (h3ndrk) [reply Readme.md]: Sure, we can do that. I also thought that there might be some embedded devices that are not connected to standard IP networking, but it is very common and therefore one of the essentials skills. Better now? 😉

File: Readme.md (hunk 3 above)
Comment (h3ndrk) [reply Readme.md:263]: Since all of us will encounter these things at one point, it might be a good idea to have a little foundation in it. 😉

**Pull request merged**